pom.xml

```diff
@@ -17,6 +17,7 @@
         <id>jitpack.io</id>
         <url>https://jitpack.io</url>
       </repository>
+
```

Comment:
why

Reply:
My IDE hates you, that's why. I'll delete